OJP/OJP_JourneySupport.xsd

```diff
@@ -191,6 +191,24 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
+	<xs:simpleType name="ReservationNeededEnumeration">
+		<xs:annotation>
+			<xs:documentation>Possible types of reservation needed for services.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="none"/>
+			<xs:enumeration value="service">
+				<xs:annotation>
+					<xs:documentation>Reservation/booking is needed that the service operates. BookingArrangement should then be used and contain more details.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="stop">
+				<xs:annotation>
+					<xs:documentation>Reservation/booking is needed for the service to touch a stop (for alighting or boading). BookingArrangement should then be used and contain more details.</xs:documentation>
```

Comment:
A reservation/booking is required for the service to call at the stop for boarding and alighting. BookingArrangement should then be used and contain more details.